README.md

```diff
@@ -97,3 +97,48 @@ Going back to the example "build" command from above, you can now run it from th
 ```sh
 $ devcmd build
 ```
+
+## Setting up a local development setup
+
+Since both the `devcmd` and the `devcmd-cli` packages use local installations within __node_modules__ folders,
+developing and especially testing devcmd locally can be challenging. In order to setup a local development
+environment, you can use the script `setup-dev`, which intializes a `dev`-folder, that contains the required `devcmd`-package
+as a yarn symlink. It is recommended to use the Remote-Containers extension for Visual Studio Code for this.
```

Comment:
and it would be even better if you could also explain how to the dev container ;)

Reply:
Adding an explanation